docs/contributor_guide/development_process.rst

```diff
@@ -133,6 +140,12 @@ Follow the steps below to create a pull request.
       * Leave a comment on the issue indicating interest and outlining possible questions.
       * Once we know you are working on it, we will support you on your contribution journey!
 
+.. note::
+
+   We would like to make sure that the issues are fairly distributed, so if you have claimed an issue and
+   haven't responded in any way in a week the issue will be free to be taken by someone else. An important distinction
```

Comment:
Pardon my non-native English, but I'm not sure what is meant with "distinction" in this sentence?

Reply:
How about "If you claim an issue, please try to keep it updated each week, either by continuing a discussion in the issue itself or in a pull request. Issues which are not receiving updates may be claimed by someone else."

Reply:
perfect @riedgar-ms thanks, added in https://github.com/fairlearn/fairlearn/pull/1486/commits/33c10615ef1fc309607a1cc42088db7c9d78d86b